src/main/java/co/novu/api/feeds/FeedsHandler.java

```diff
@@ -1,33 +1,39 @@
 package co.novu.api.feeds;
 
+import java.io.IOException;
+
 import co.novu.api.feeds.request.FeedRequest;
-import co.novu.api.feeds.response.FeedResponse;
 import co.novu.api.feeds.response.BulkFeedsResponse;
-import co.novu.common.base.NovuConfig;
+import co.novu.api.feeds.response.FeedResponse;
+import co.novu.common.rest.NovuNetworkException;
 import co.novu.common.rest.RestHandler;
-import lombok.RequiredArgsConstructor;
+import retrofit2.Response;
 
-@RequiredArgsConstructor
 public class FeedsHandler {
 
     private final RestHandler restHandler;
+    
+    private final FeedsApi feedsApi;
+    
+    public FeedsHandler(RestHandler restHandler) {
```

Comment:
You'd need to modify the constructor of FeedsHandler in Novu.java class. Please check that